spec/system/courses/students_spec.rb

```diff
@@ -140,6 +178,12 @@
     expect(page).to have_text(team_3.name)
   end
 
+  scenario 'coach checks the distribution of students by level' do
+    sign_in_user team_coach.user, referer: students_course_path(course)
+
+    expect(page).to have_text('L1')
+  end
```

Comment:
I am not sure what this is doing here when there is a much to exhaustive check in scenarios above this. 🤔